documentation/examples/documented/src/main/scala/counter/Counter.scala

```diff
@@ -54,54 +56,64 @@ trait CounterEndpoints
   // that they return an HTTP response whose entity contains a JSON document
   // with the counter value
   lazy val counterJson =
-    jsonResponse[Counter](documentation = "The counter current value")
+    jsonResponse[Counter](docs = Some("The counter current value"))
 
-  // We generically derive a data type schema. This schema
+  // We require json schema for `Counter` from interpreters. This schema
   // describes that the case class `Counter` has one field
   // of type `Int` named “value”
-  implicit lazy val jsonSchemaCounter: JsonSchema[Counter] = genericJsonSchema
+  implicit def jsonSchemaCounter: JsonSchema[Counter]
```

Comment:
Do you have an idea how to fix it? We cannot reuse the schema interpretation because in openapi its real schema and in server its circe codec generated via schema api. To implement desired behaviour we would need an ability to derive circe codec from already materialized openapi schema.

Reply:
> To implement desired behaviour we would need an ability to derive circe codec from already materialized openapi schema.

Yes, we used to have such an interpreter: https://static.javadoc.io/org.julienrf/endpoints-openapi-json-schema-circe_2.12/0.4.0/endpoints/documented/circe/JsonSchemas.html

Reply:
Wow, must have missed it, certainly need to fix it.